### DISCUSSION TITLE
PSP version straight up won't work.

### DISCUSSION THREAD

**Discussion by mazemuster7** [CrossCraft/CrossCraft-SurvivalTest#10] [Q&A]
It opens, but once I try to actually play it, it just boots me back to the home screen.

**Comment by IridescentRose** [comment]
Try checking the Installation instructions provided with the release -- make sure you extracted the assets folder correctly.

**Reply by mazemuster7** [reply]
> Try checking the Installation instructions provided with the release -- make sure you extracted the assets folder correctly.

There are none. I've checked.
